test/conftest.py

```diff
@@ -54,7 +54,7 @@ def pytest_generate_tests(metafunc):
     # @pytest.mark.parametrize("document_store", ["memory"], indirect=False)
     found_mark_parametrize_document_store = False
     for marker in metafunc.definition.iter_markers('parametrize'):
-        if 'document_store' in marker.args[0] or 'document_store_with_docs' in marker.args[0] or 'document_store_type' in marker.args[0]:
```

Comment:
I am still wondering why we had these additional checks if `marker.args[0]` is just a string and `document_store` is a substring of `document_store_with_docs`, etc. However, the fix looks good to me.

Reply:
I suppose when they were just added it was assumed that marker.args[0] is a list because `in` is used. The actual reason for using `in` is, however, that the string can include a combination with a comma like `document_store,retriever`.